src/org/robockets/steamworks/DrivePodOutput.java

```diff
@@ -0,0 +1,29 @@
+package org.robockets.steamworks;
+
+import edu.wpi.first.wpilibj.PIDOutput;
+import edu.wpi.first.wpilibj.SpeedController;
+
+/**
+ * PIDOutput used for drivepods
+ */
+public class DrivePodOutput implements PIDOutput {
+
+	private SpeedController c1, c2;
+
+	/**
+	 *
+	 * @param c1 Left side speed controller
+	 * @param c2 Right side speed controller
```

Comment:
Shouldn't these say front and back? Not that it matters, since there's only one speed controller now.